packages/common/src/ticker.ts

```diff
@@ -59,13 +59,20 @@ export class Ticker {
     let ticker = document.createElement("div");
     ticker.classList.add("en__component");
     ticker.classList.add("en__component--ticker");
-    let str = '<div class="ticker">';
+    let str = `<div class="ticker">`;
```

Comment:
I don't see a problem with this change, but I would like to understand the reason :)

Reply:
Oh I can actually change this back, I was originally using a variable in the string in the ${variable} style but I removed it.